code/game/objects/items/grenades/syndieminibomb.dm

```diff
@@ -59,11 +59,11 @@
 	update_mob()
 	playsound(loc, 'sound/effects/empulse.ogg', 50, TRUE)
 	radiation_pulse(src, rad_damage)
-	for(var/turf/T in view(freeze_range,loc))
-		if(isfloorturf(T))
-			var/turf/open/floor/F = T
-			F.MakeSlippery(TURF_WET_PERMAFROST, 6 MINUTES)
-			for(var/mob/living/carbon/L in T)
-				L.adjustStaminaLoss(stamina_damage)
-				L.adjust_bodytemperature(-230)
+	for(var/turf/turf_affected in view(freeze_range,loc))
+		if(isfloorturf(turf_affected))
```

Comment:
Last thing, didn't notice this until now--sorry, chang this to `for (var/turf/open/floor/floor in view(freeze_range, loc))`.

If we're performing a type-check anyway, we might as well do it in the definition.